docs/data/additional_components.yml

```diff
@@ -33,12 +39,6 @@ flink-shaded-17.0:
   source_release_asc_url: "https://downloads.apache.org/flink/flink-shaded-17.0/flink-shaded-17.0-src.tgz.asc"
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-shaded-17.0/flink-shaded-17.0-src.tgz.sha512"
 
-flink-shaded-16.2:
-  name: "Apache Flink-shaded 16.2 Source Release"
-  source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-shaded-16.2/flink-shaded-16.2-src.tgz"
-  source_release_asc_url: "https://downloads.apache.org/flink/flink-shaded-16.2/flink-shaded-16.2-src.tgz.asc"
-  source_release_sha512_url: "https://downloads.apache.org/flink/flink-shaded-16.2/flink-shaded-16.2-src.tgz.sha512"
-
```

Comment:
Do we only keep the latest version of shaded, or do we keep all of them?

Reply:
I believe we keep 3 latest versions here and all in the archives.